test/kpod_pause.bats

```diff
@@ -4,6 +4,10 @@ load helpers
 
 IMAGE="redis:alpine"
 
+function setup() {
+    copy_images
```

Comment:
We only use the `redis:alpine` image in this file, so this probably doesn't speed it up.